src/main/interlok/config/variables.properties

```diff
@@ -5,4 +5,9 @@ adapter.base.url=file\://localhost/${user.dir}
 
 hpcc.server=http://${HPCC_HOST}:8010
 source.ip=${INTERLOK_HOST}:7100
-dfuplus.cmd=/usr/bin/dfuplus
\ No newline at end of file
+dfuplus.cmd=/usr/bin/dfuplus
+
+amazon.access.key=${AWS_ACCESS_KEY}
+amazon.secret.key=${AWS_SECRET_KEY}
```

Comment:
Should we switch to using localstack to so that we don't need secrets?

Reply:
Well the old behaviour was to use real AWS. I think it's a valid issue to raise, but not as part of this since it's going to be a docker compise file change, additional docker image, and all sorts.